CHANGELOG.md

```diff
@@ -1,13 +1,19 @@
 # Changelog
 
-## ? - WIP
-- Configurable HTML and AMP element exporters
+## 0.2.0 - 2017/11/02
```

Comment:
wrong date :)

Reply:
Oups 😄 I've prepared the PR yesterday and forgot to change the date 🙈